datadog_checks_base/datadog_checks/base/stubs/tagging.py

```diff
@@ -0,0 +1,40 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+class TaggerStub(object):
+    LOW, ORCHESTRATOR, HIGH = range(3)
+
+    def __init__(self):
+        self.reset()
+
+    def reset(self):
```

Comment:
Could we add some `docstrings` to document these and mark which are use for test purposes?